modules/ossm-migrating-reference-unsupported-fields.adoc

```diff
@@ -0,0 +1,131 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/reference/ossm-migrating-reference.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-migrating-reference-unsupported-fields_{context}"]
+= Unsupported fields in Service Mesh 3
+
+The following tables list fields that are no longer supported in {SMProductName} 3. Remember that this does not necessarily mean the functionality has been removed. In many cases, it means that default values are being used instead.
+
+== Add on Configurations
+
+Remember that add ons, such as {DTProductName}, {KialiProduct}, and others, are managed and configured separately in {SMProduct} 3. See "Observability and Service Mesh" for more information on how to configure them.
+
+[cols="1"]
+|===
+| spec.addons.3scale
+| spec.addons.grafana
+| spec.addons.jaeger
+| spec.addons.kiali
+| spec.addons.prometheus
+| spec.addons.stackdriver
+|===
+
+//For details on how to enable addon integrations, see [Observability Integration](https://docs.redhat.com/en/documentation/red_hat_openshift_service_mesh/3.0.0tp1/html/observability/index).
+
+== Cluster configurations
+
+[cols="1"]
+|===
+| cluster.meshExpansion.ilbGateway,
+| cluster.multiCluster.meshNetworks.gateways.service
+|===
+
+
+== Gateways configurations
+
+Gateways are managed separately in {SMProduct} 3.
+
+== Policy configurations
+
+[cols="1,1"]
+|===
+| SMCP 2.6 configuration | Istio 3.0 configuration
+| spec.policy.type
+| No direct equivalent (Istiod is now the only option)
+| spec.policy.mixer
+| Removed (Mixer deprecated)
+| spec.policy.remote
+| Removed (Remote policy checking replaced)
+|===
+
+== Proxy configurations
+
+=== Proxy networking configuration fields
+
+[cols="1"]
+|===
+| proxy.networking.initialization.type
+| proxy.networking.initialization.initContainer.runtime.env
+| proxy.networking.protocol.autoDetect
+| proxy.networking.protocol.inbound
+| proxy.networking.protocol.outbound
+|===
+
+== Runtime configurations
+
+=== Deployment configuration fields
+
+[cols="1,"]
+|===
+| runtime.components.deployment.strategy.type
+| Not directly available
```

Comment:
Somewhere we use 'Not directly available' and somewhere just empty column. We should unify that.